app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -93,6 +93,16 @@ class MainActivity : AppCompatActivity() {
     private var lastSpeed: Float = 0f
     private var calibrationProgress = 0f
 
+    private var sumSegmentationDetectionFPS = 0f
```

Comment:
Mind extracting this logic to some custom view or just class that handles the logic? Will be able to reuse this component than in other activities as well, we'll definitely need this on AR as well

Reply:
will do

Reply:
Lets also add calibration progress and and current session time to the panel?